bakery_lint/fonttests/test_ttf.py

```diff
@@ -632,14 +634,16 @@ def test_check_panose_identification(self):
     @tags('note')
     @autofix('bakery_cli.fixers.AddSPUAByGlyphIDToCmap')
     def test_font_unencoded_glyphs(self):
-        """ Font does not have unencoded glyphs """
+        """ Is there any unencoded glyph? """
         ttx = ttLib.TTFont(self.operator.path, 0)
         unencoded_glyphs = get_unencoded_glyphs(ttx)
+        #TODO: Shouldn't we explicitely mention here
+        #      which ones are the unencoded glyphs ?
```

Comment:
@felipesanches yes, we should, but rather than writing TODOs it would be better to file issues to discuss such topics :)